loadgen_pymodule_setup_src.py

```diff
@@ -0,0 +1,27 @@
+"""MLPerf Inference LoadGen python bindings.
+
+Creates a module that python can import.
+All source files are compiled by python's C++ toolchain  without depending
+on a loadgen lib.
+"""
+
+from setuptools import setup, Extension
+
+sources = [
+  "bindings/c_api.cc",
+  "bindings/python_api.cc",
+  "parse_command_line.cc",
+  "loadgen.cc",
+]
+
+mlpi_loadgen_module = Extension('mlpi_loadgen',
+                    define_macros = [('MAJOR_VERSION', '0'),
+                                     ('MINOR_VERSION', '5')],
+                    include_dirs = [ 'gen' ],
```

Comment:
is there a directory gen?

Reply:
It's created by the gn metabuild and holds generated files. In this case, copied source files that it uses to generate the wheel python module. I need to figure out how to do it without copying the source to the gen directory.

Reply:
I added a standalone setup.py that doesn't need gn or ninja and goes ahead and pollutes the source directory.